### PR TITLE
[Dependency Update]this pr apply latest neo to devpack

### PR DESCRIPTION
**Pull request by Jim8y**
This pr updates neo and apply latest neo change.

### CODE REVIEW

File: tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs [hunk 1]
```diff
@@ -19,11 +20,11 @@ public abstract class MyUndeployedContract : SmartContract
         }
 
         //[TestMethod]
-        public void GenerateNativeArtifacts()
+        public void GenerateNativeArtifacts(DataCache snapshot)
         {
             foreach (var n in Neo.SmartContract.Native.NativeContract.Contracts)
             {
-                var manifest = n.Manifest;
+                var manifest = Neo.SmartContract.Native.NativeContract.ContractManagement.GetContract(snapshot, n.Hash).Manifest;
```
Comment (shargon) [inline tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs:27]: #947 this solution require the initialization

Comment (Jim8y) [reply tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs:27]: cant get any clue

Comment (shargon) [reply tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs:27]: Wrong pr hahaha

File: tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs [hunk 2]
```diff
@@ -19,11 +20,11 @@ public abstract class MyUndeployedContract : SmartContract
         }
 
         //[TestMethod]
-        public void GenerateNativeArtifacts()
+        public void GenerateNativeArtifacts(DataCache snapshot)
```
Comment (shargon) [inline tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs:23]:
```suggestion
        public void GenerateNativeArtifacts()
```

File: tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs [hunk 3]
```diff
@@ -6,6 +6,7 @@
 using Neo.VM.Types;
 using System.Collections.Generic;
 using System.IO;
+using Neo.Persistence;
```
Comment (shargon) [inline tests/Neo.SmartContract.Testing.UnitTests/TestEngineTests.cs:9]:
```suggestion
```

**Pull request merged**